lib/utilities/process-template.js

```diff
@@ -0,0 +1,10 @@
+var _template = require('lodash.template');
```

Comment:
Eh, by the time you've required the `processTemplate` helper you're gonna need it. 😜

I really truly don't care about this, so somebody tell me what to do! (Too many people with opinions.)